redis_arginfo.h

```diff
@@ -1433,3 +1433,30 @@ static const zend_function_entry class_Redis_methods[] = {
 	ZEND_ME(Redis, zunionstore, arginfo_class_Redis_zunionstore, ZEND_ACC_PUBLIC)
 	ZEND_FE_END
 };
+
+
+static const zend_function_entry class_RedisException_methods[] = {
+	ZEND_FE_END
+};
+
+static zend_class_entry *register_class_Redis(void)
+{
+	zend_class_entry ce, *class_entry;
+
+	INIT_CLASS_ENTRY(ce, "Redis", class_Redis_methods);
+	class_entry = zend_register_internal_class_ex(&ce, NULL);
+
+	zend_mark_function_parameter_as_sensitive(&class_entry->function_table, "auth", 0);
```

Comment:
Do we also need to put `__construct` here?

Reply:
Good question ;)

`$options` is not really sensible, but may contains `auth` entry which is sensible.

So... don't know, as you prefer